plugin/reload/reload.go

```diff
@@ -29,7 +60,7 @@ func hook(event caddy.EventName, info interface{}) error {
 	// if reload is removed from the Corefile, then the hook
 	// is still registered but setup is never called again
 	// so we need a flag to tell us not to reload
-	if r.usage == unused {
+	if r.getUsage() == unused {
```

Comment:
The "idiomatic" way of naming getters in Go is not to use "get" in the name.   e.g. `r.Usage()` ...

Reply:
Thanks, @chrisohaver.